.circleci/config.yml

```diff
@@ -998,6 +997,37 @@ jobs:
             deploy_ecr aztec-sandbox
             deploy_dockerhub aztec-sandbox
 
+  deploy-end:
+    docker:
+      - image: cimg/base:current
+    resource_class: small
+    steps:
+      - run:
+          name: "Noop"
+          command: echo Noop
+
+  build-canary:
+    machine:
+      image: ubuntu-2004:202010-01
+    resource_class: large
+    steps:
+      - *checkout
+      - *setup_env
+      - run:
+          name: "Build and test"
+          command: build canary true
+
+  run-canary:
+    docker:
+      - image: aztecprotocol/alpine-build-image
+    resource_class: small
+    steps:
+      - *checkout
+      - *setup_env
+      - run:
+          name: "Test"
+          command: spot_run_tests canary uniswap_trade_on_l1_from_l2.test.ts docker-compose.yml
```

Comment:
Are these happening on every build? Or should we set them up to run only on tagged releases?